### PR TITLE
React components

### PR DESCRIPTION
**Pull request by DiemBTran**
We created the edit functionality with Aaron. We want to work on new functionality on a new branch.

### CODE REVIEW

File: app/javascript/components/mockEmergencyContact.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        phone_number: "1234567890",
+        user_id: 2
+    },
+    {
+        id: 4,
+        full_name: "Shikamaru Nara",
+        relationship: "proctor",
+        phone_number: "1234567890",
+        user_id: 2
+    },
```
Comment (Randonee) [inline app/javascript/components/mockEmergencyContact.js]: nitpick: It looks like trailing camas are not used in other places so this one could be removed.

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -1,65 +1,87 @@
...
+            render={(props) => <UserProfile personalInfo={this.state.personalInfo} />}
+          />*/}
+
+          {/*Not Found*/}
+          <Route component={NotFound} />
+
+          { /* Protected Routes */}
+        </Switch>
+        <Home />
+        <LearnMore />
```
Comment (Randonee) [inline app/javascript/components/App.js]: I think ```<Home />``` and ```<LearnMore />``` were supposed to be removed. The ```<Route>``` tags below also need to be inside the ```<switch />```
The Router should have the form: 
```
<Router>
  <Switch>
    <Route />
    <Route />
  </Switch>
</Router>
```

**Review by Randonee — CHANGES_REQUESTED** [review]
The router has a few things out of place. See comments

File: app/javascript/components/App.js (hunk 2 above)
Comment (bndiep) [reply app/javascript/components/App.js]: I see what you mean. It looks like the App.js component that you're looking at is outdated. On our local machines, when we pull directly from the branch react-components, the changes that you have suggested have already been made.

**Review by Randonee — APPROVED** [review]
Nice!, LGTM

**Pull request merged**